File: server/src/platforms/Vimeo.ts
import { PlaylistCapableBasePlatform } from "./PlaylistCapableBasePlatform.js";

export class Vimeo extends PlaylistCapableBasePlatform {
  readonly id = "vimeo";

  static readonly HOSTS = ["vimeo.com", "www.vimeo.com", "player.vimeo.com"];
}

File: app/modules/ytdlp/android/src/main/java/expo/modules/ytdlp/platforms/Vimeo.kt
package expo.modules.ytdlp.platforms

class Vimeo(url: String) : PlaylistCapableBasePlatform(url) {
    companion object {
        val HOSTS = setOf("vimeo.com", "www.vimeo.com", "player.vimeo.com")
    }
}
